### PR TITLE
Cmdliner

### PR DESCRIPTION
**Pull request by gasche**
Below is the rendered output of `dune exec -- mustache --help` after this PR.

```man
MUSTACHE(1)                     Mustache Manual                    MUSTACHE(1)



NAME
       mustache - renders Mustache template from JSON data files

SYNOPSIS
       mustache [OPTION]... DATA.json TEMPLATE.mustache

DESCRIPTION
       mustache is a command-line tool coming with the ocaml-mustache library,
       an OCaml implementation of the Mustache template format. mustache takes
       a data file and a template file as command-line parameters; it renders
       the populated template on standard output.

       Mustache is a simple and popular template format, with library
       implementations in many programming languages. It is named from its
       {{..}} delimiters.

       Mustache website
           https://mustache.github.io/

       Mustache templates documentation
           https://mustache.github.io/mustache.5.html

       ocaml-mustache website:
           https://github.com/rgrinberg/ocaml-mustache

       The ocaml-mustache implementation is tested against the Mustache
       specification testsuite. All features are supported, except for lambdas
       and setting delimiter tags.

ARGUMENTS
       DATA.json (required)
           data file in JSON format

       TEMPLATE.mustache (required)
           mustache template

OPTIONS
       --help[=FMT] (default=auto)
           Show this help in format FMT. The value FMT must be one of `auto',
           `pager', `groff' or `plain'. With `auto', the format is `pager` or
           `plain' whenever the TERM env var is `dumb' or undefined.

EXAMPLES
       $ cat data.json
       { "name": "OCaml",
         "qualities": [{"name": "simple"}, {"name": "fun"}] }

       $ cat hello.mustache
       Hello {{name}}!
       Mustache is:
       {{#qualities}}
       - {{name}}
       {{/qualities}}

       $ mustache data.json hello.mustache
       Hello OCaml!
       Mustache is:
       - simple
       - fun


BUGS
       Report bugs on https://github.com/rgrinberg/ocaml-mustache/issues



Mustache                                                           MUSTACHE(1)
```

### CODE REVIEW

File: bin/test/errors/json-errors.t [hunk 1]
```diff
@@ -4,10 +4,14 @@
 
 Empty json file:
   $ mustache empty.json foo.mustache
-  Fatal error: exception Ezjsonm.Parse_error(870828711, "JSON.of_buffer expected JSON text (JSON value)")
-  [2]
+  mustache: internal error, uncaught exception:
+            Ezjsonm.Parse_error(870828711, "JSON.of_buffer expected JSON text (JSON value)")
```
Comment (rgrinberg) [inline bin/test/errors/json-errors.t:8]: It's not required for this PR, but it would be nice to pretty print these exceptions.

**Review by rgrinberg — APPROVED** [review]

**Comment by rgrinberg** [comment]
Installing the manpage would be nice as well:

```
(rule
 (with-stdout-to mustache.1
  (run mustache --help=groff)))

(install
 (section man)
 (package mustache)
 (files mustache.1))
```

File: bin/test/errors/json-errors.t (hunk 1 above)
Comment (gasche) [reply bin/test/errors/json-errors.t:8]: It's on my mind. I think that giving nice error messages for those would require getting the error location in the JSON file, which may require an upstream improvement to Ezjsonm or the use of Jsonm directly.

**Comment by gasche** [comment]
Thanks! I added the manpage and merged.

**Pull request merged**